/**
 * Created with IntelliJ IDEA.
 * User: glukki
 * Date: 09.05.12
 * Time: 16:28
 * To change this template use File | Settings | File Templates.
 */

import "Flowerbed.proto";
option java_package = "com.kloomba.app.api.protobuf";
option java_outer_classname = "BookmarkListProto";

//request: /bookmark/list
message BookmarkList {
    optional int32 timestamp = 1;
    repeated Flowerbed flowerbed = 2;  //bookmark list
}